app/views/product-pages/signup/get-started.html

```diff
@@ -0,0 +1,129 @@
+            ></path>
+          </svg>
+          <span class="govuk-header__logotype-text"> GOV.UK </span>
+        </span>
+        <span class="govuk-header__product-name">
+          Forms
+          <strong class="govuk-tag govuk-phase-banner__content__tag">
+            beta
+          </strong>
+        </span>
+      </a>
+    </div>
+    <div class="govuk-header__content">
+      <nav aria-label="Menu" class="govuk-header__navigation">
+        <button
+          type="button"
+          class="govuk-header__menu-button govuk-js-header-toggle"
+          aria-controls="navigation"
+          aria-label="Show or hide menu"
+          hidden=""
+        >
+          Menu
+        </button>
+
+          <li class="govuk-header__navigation-item govuk-header__navigation-item--active">
+            <a class="govuk-header__link" href="get-started"> Get started </a>
+          </li>
+          <li class="govuk-header__navigation-item" >
+            <a class="govuk-header__link" href="features"> Features </a>
+          </li>
+          <li class="govuk-header__navigation-item" >
+            <a class="govuk-header__link" href="support"> Support </a>
+          </li>
+          <li class="govuk-header__navigation-item" >
+            <a class="govuk-header__link" href="https://admin.research.forms.service.gov.uk/"> Sign in </a>
+          </li>
+        </ul>
+      </nav>
+    </div>
+  </div>
+</header>
+{% endblock header %}
+
+{% block content %}
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">Get started</h1>
+    <p class="govuk-body">
+      GOV.UK Forms is a new platform that makes it easy to create accessible
+      online forms for GOV.UK.
+    </p>
+
+    <ol class="task-list">
+      <li class="task-list__item">
+        <h2 class="task-list__heading"><span class="task-list-section-number">1.</span> Check if GOV.UK Forms is right for you</h2>
+        <div class="task-list__details">
+          <p class="govuk-body">
+            You can use GOV.UK Forms if you’re from a central government organisation
+            and need to collect information from people for a government service.
+          </p>
+          <p class="govuk-body">
+            Find out more about the <a class="govuk-link" href="features">features of GOV.UK Forms</a>.
+          </p>
+        </div>
+      </li>
+
+      <li class="task-list__item">
+        <h2 class="task-list__heading"><span class="task-list-section-number">2.</span> Create a trial account</h2>
+        <div class="task-list__details">
+          <p class="govuk-body">
+            If you want to try it out you can create a trial account.
+          </p>
+          <p class="govuk-body">You’ll need your government email address.</p>
+          <p class="govuk-body">
+            <a class="govuk-button govuk-button--start" href="https://admin.research.forms.service.gov.uk/signup" >Create a trial account</a >
```

Comment:
I've created this route now in alphagov/forms-admin#532, only thing is I called it `/sign-up` (to match `sign-out`) and it only makes a difference for Auth0 (for CDDO SSO it just goes to the normal sign in page)

Reply:
I changed the link in the prototype